assets/terraform/aws/bootstrap.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
I'll make all TF template enhancements part of a separate PR after I test all deployment params such as different distros, etc; don't hold this specific PR for too long.